### PR TITLE
FIX Remove Context Broker specific setup in RPM

### PR DESCRIPTION
**Pull request by dmoranj**
Remove Context Broker specific setup from the RPM Specs.

### CODE REVIEW

File: rpm/SPECS/pepProxy.spec [hunk 1]
```diff
@@ -90,18 +90,6 @@ echo "[INFO] Configuring application"
     setfacl -d -m g::rwx %{_pepProxy_log_dir}
```
Comment (fgalan) [inline rpm/SPECS/pepProxy.spec]: Maybe documentation should be reviewed, e.g. it seems that the following text is no longer valid after implementing this change:

"If the PEP Proxy is deployed in a machine with an installed Context Broker service, the PEP Proxy will automatically change the Context Broker port to the 10026 and install itself on the port where the Context Broker was listening, so no further configuration should be needed for the connectivity."

Probably there are more documentation tokens to be adjusted (I haven't checked it in long).

File: rpm/SPECS/pepProxy.spec [hunk 2]
```diff
@@ -90,18 +90,6 @@ echo "[INFO] Configuring application"
...
     setfacl -d -m o::rx %{_pepProxy_log_dir}
 
-    echo "[INFO] Checking Context Broker installations"
```
Comment (fgalan) [inline rpm/SPECS/pepProxy.spec]: CHANGES_NEXT_RELEASE entry needed? It is a change in the way the RPM behaves, it makes sense to me mention it in the changelog buffer file.

Comment (dmoranj) [inline rpm/SPECS/pepProxy.spec]: Fixed in f89655c

File: rpm/SPECS/pepProxy.spec (hunk 1 above)
Comment (dmoranj) [inline rpm/SPECS/pepProxy.spec]: Fixed in f89655c

**Comment by fgalan** [comment]
LGTM (waiting for @jcalderin to do the final LGTM + merge)

**Comment by dmoranj** [comment]
Fixes #159

**Comment by jcalderin** [comment]
LGTM

**Pull request merged**